<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditWindow</class>
 <widget class="QDialog" name="EditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EditWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="textId">
     <property name="text">
      <string>Id</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="textName">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editEmail">
     <property name="placeholderText">
      <string>Email</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="textYear">
     <property name="text">
      <string>Year</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editThesis">
     <property name="placeholderText">
      <string>Thesis</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="textCoordinator">
     <property name="text">
      <string>Coordinator</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonGroup">
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
